apis/metal3.io/v1alpha1/baremetalhost_types.go

```diff
@@ -442,6 +442,18 @@ const (
 	TeraByte          = GigaByte * 1000
 )
 
+// DiskType is a disk type, i.e. HDD, SSD, NVME.
+type DiskType string
+
+// DiskType constants.
+const (
+	UNKNOWN DiskType = ""
```

Comment:
This value doesn't match the enum values in the CRD.

Reply:
Not following, why?

![image](https://user-images.githubusercontent.com/11349489/120547438-cab40300-c3f9-11eb-8738-6c0e6fbc6037.png)

Reply:
Because the value here is `""`, not `"UNKNOWN"`.

Reply:
ohh, yea, obviously, thanks!

Reply:
fixed